src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -155,10 +172,39 @@ private MongoConnection createConnection(ConnectionString cs, Properties info)
             throws SQLException {
         // attempt to get DIALECT property, and default to "mysql" if none is present
         String dialect = info.getProperty(DIALECT, MYSQL_DIALECT);
+        // Default log level is OFF
+        String logLevelVal = info.getProperty(LOG_LEVEL, Level.OFF.getName());
+        Level logLevel;
+        try {
+            logLevel = Level.parse(logLevelVal.toUpperCase());
+        } catch (IllegalArgumentException e) {
+            throw new SQLException(
+                    "Invalid "
+                            + LOG_LEVEL
+                            + " property value : "
+                            + logLevelVal
+                            + ". Valid values are : "
+                            + LEVELS
+                            + ".");
+        }
+        String logDirVal = info.getProperty(LOG_DIR);
+        File logDir = (logDirVal == null) ? null : new File(logDirVal);
```

Comment:
I'm a little bit on the fence on this, despite writing the behavior in the design doc, because if the directory does not exist and the user doesn't have "access" to stdout (because the driver is used in a tool which does not redirect stdout for example), then the fact that the driver won't log in the provided directory will be lost, possibly confusing users and making things for everyone hard to debug.

Reply:
@bucaojit Let's discuss this tomorrow during stand-up

Reply:
Yes, that makes sense not to default to stdout if the directory does not exist as the user did set the property and would expect the logs to be there.  What I think would be nice to be added is a way to specify "print to stdout" using the LogDir property versus the lack of the property being what sets the behavior.

Reply:
I made the change for this.  If no value specified for LogDir or LogDir=console, it will print to the console.